www/js/diary/services.js

```diff
@@ -1,22 +1,23 @@
 'use strict';
 
 const MotionTypes = {
-  IN_VEHICLE: {name: "IN_VEHICLE", icon: "ion-speedometer", color: "purple"},
-  BICYCLING: {name: "BICYCLING", icon: "ion-android-bicycle", color: "green"},
-  UNKNOWN: {name: "UNKNOWN", icon: "ion-ios-help", color: "orange"},
-  WALKING: {name: "WALKING", icon: "ion-android-walk", color: "brown"},
-  RUNNING: {name: "RUNNING", icon: "ion-android-walk", color: "brown"}, // TODO: do we need this or is it always converted to WALKING?
-  CAR: {name: "CAR", icon: "ion-android-car", color: "red"},
-  AIR_OR_HSR: {name: "AIR_OR_HSR", icon: "ion-plane", color: "red"},
+  IN_VEHICLE: {name: "IN_VEHICLE", icon: "speedometer", color: "purple"},
+  ON_FOOT: {name: "ON_FOOT", icon: "walk", color: "brown"},
+  BICYCLING: {name: "BICYCLING", icon: "bike", color: "green"},
+  UNKNOWN: {name: "UNKNOWN", icon: "help", color: "orange"},
+  WALKING: {name: "WALKING", icon: "walk", color: "brown"},
+  RUNNING: {name: "RUNNING", icon: "walk", color: "brown"}, // TODO: do we need this or is it always converted to WALKING?
```

Comment:
I think we convert it to WALKING

Reply:
I think that we also need an `ON_FOOT` just in case we end up getting cleaned trips instead of inferred trips

Reply:
ON_FOOT is there; I experienced that when testing without an inference model and receiving cleaned sections

Reply:
*TODO later* (remove RUNNING)

Reply:
removed in #1000